File: dbflow/src/androidTest/java/com/raizlabs/android/dbflow/test/structure/ForeignParentModel.java
package com.raizlabs.android.dbflow.test.structure;

import com.raizlabs.android.dbflow.annotation.Table;
import com.raizlabs.android.dbflow.test.TestDatabase;

/**
 * Description:
 */
@Table(database = TestDatabase.class)
public class ForeignParentModel extends TestModel1 {
}

File: dbflow-tests/src/test/java/com/raizlabs/android/dbflow/test/structure/foreignkey/ForeignParentModel.kt
package com.raizlabs.android.dbflow.test.structure.foreignkey

import com.raizlabs.android.dbflow.annotation.Table
import com.raizlabs.android.dbflow.test.TestDatabase
import com.raizlabs.android.dbflow.test.structure.TestModel1

/**
 * Description:
 */
@Table(database = TestDatabase::class)
open class ForeignParentModel : TestModel1()
